### ISSUE TITLE
VS DataContext Instructions out of date in section "Setting DataContext in markup"

### ISSUE THREAD

**Issue by jraldrin** [MicrosoftDocs/windows-uwp#1399]
I'm using VS2017 version 15.9.5 and there is no "New" Button associated with the Data Context Property.  Also the only items in enabled in little square button next to "Data Context" are "Custom Expression" and "System Resource"

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 51100eeb-4a55-5997-eee8-2ed0047be78f
* Version Independent ID: d9d5ad3b-4932-c02f-bac0-4dd5a9dd48fb
* Content: [Sample data on the design surface, and for prototyping - Windows UWP applications](https://docs.microsoft.com/en-us/windows/uwp/data-binding/displaying-data-in-the-designer)
* Content Source: [windows-apps-src/data-binding/displaying-data-in-the-designer.md](https://github.com/MicrosoftDocs/windows-uwp/blob/docs/windows-apps-src/data-binding/displaying-data-in-the-designer.md)
* Product: **uwp**
* GitHub Login: @mcleanbyron
* Microsoft Alias: **mcleans**

**Comment by GrantMeStrength** [comment]
Thanks for the comment, and sorry for the delay in addressing it. 

I have opened the project, and can confirm there is no new button as expect. I have reached out to the author of the sample to see if it requires updating.

Thanks for taking the time to leave feedback, and again, sorry for the delay. We do read bugs as they come in, but may not add comments immediately, which gives a poor impression - I apologize. -John
